src/Geometry/Primitives/NonCSG/ConvexPolygon.jl

```diff
@@ -168,11 +167,11 @@ end
 
 Create a triangle mesh that can be rendered by iterating on the polygon's edges and for each edge use the centroid as the third vertex of the triangle.
 """
-function makemesh(poly::ConvexPolygon{N,T}, ::Int = 0) where {N,T<:Real}
+function makemesh(poly::ConvexPolygon{T,N}, ::Int = 0) where {N,T<:Real}
     c = centroid(poly)
 
     l2w = local2world(poly.local_frame)
-    len = length(poly.local_points)
+    len = size(poly.local_points)[2] 
```

Comment:
Is this right? local_points is a Vector{SVector{2,T}}. 
```
size(poly.local_points)[2]
```
gives an indexing error on my machine.

Reply:
will check.

Reply:
poly.local_points is an SMatrix - that is why i changed it from length to size